jsplot/pom.xml

```diff
@@ -90,6 +90,10 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-core</artifactId>
```

Comment:
you could probably remove this since it will get pulled in by `jackson-databind`